plugins/woocommerce-admin/client/products/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const NUMBERS_AND_ALLOWED_CHARS = '[^-0-9%s1%s2]';
+export const NUMBERS_AND_DECIMAL_SEPARATOR = '[^-\\d\\%s]+';
+export const ONLY_ONE_DECIMAL_SEPARATOR = '[%s](?=%s*[%s])';
+// This should never be a real slug value of any existing shipping class
+export const ADD_NEW_SHIPPING_CLASS_OPTION_VALUE =
+	'__ADD_NEW_SHIPPING_CLASS_OPTION__';
+export const UNCATEGORIZED_CATEGORY_SLUG = 'uncategorized';
```

Comment:
Yes, that's interesting! I'm wondering what is the logic WP is using to assign this category by default to any product if the user never select any category at all. If this is a variable thing then how do they deal with this? If you can enlighten me with that answer will be great for me, otherwise I think I should find the answer by myself.
Anyway this is a super great catch!

Reply:
Looks like the default category is stored in an option - `get_option( 'default_product_cat', 0 )`.

We should probably rely on that instead of the slug/name since those could be changed.

This is obviously unrelated to this PR though, so I don't want to block your work here.  Would you mind creating a follow-up issue for this?

Reply:
https://github.com/woocommerce/woocommerce/issues/35158